### ISSUE TITLE
FTLCONF_dns_upstream recommendation not correct in log

### ISSUE THREAD

**Issue by yfhyou** [pi-hole/docker-pi-hole#1564]
This is a: bug

## Details

Docker suggests to set `FTLCONF_dns_upstream` in the log, however the actual variable is `FTLCONF_dns_upstreams` per the API documentation.

```
  [i] Starting docker specific checks & setup for docker pihole/pihole

  [i] Ensuring basic configuration by re-running select functions from basic-install.sh

  [X] No DNS upstream servers are set!
  [i] Recommended: Set the upstream DNS servers in the environment variable FTLCONF_dns_upstream
```

**Comment by rdwebdesign** [comment]
This is just a typo and there is already a PR to fix it: https://github.com/pi-hole/docker-pi-hole/pull/1557

**Comment by PromoFaux** [comment]
Merged #1557 - should be fixed in the next build (I can't remember if it builds immediately on merge to `development-v6` or if you'll need to wait for the nightly...)
